cern_access/indico_cern_access/plugin.py

```diff
@@ -227,7 +227,9 @@ def _registration_form_validated(self, form, **kwargs):
 
         required = req.data.get('regform_data_mode') == RegformDataMode.during_registration_required
         form_cls = AccessIdentityDataForm if required else RegistrationFormPersonalDataForm
-        g.personal_data_form = form = form_cls()
+        # we pass 'csrf_enabled' on to allow unit tests to set it to False
+        g.personal_data_form = form = form_cls(csrf_enabled=form.meta.csrf)
```

Comment:
i think it could always be disabled here since 1) we have a global csrf check and 2) the regform probably has a csrf check as well

Reply:
Does it hurt to leave it like this?

Reply:
nope